python/lsst/ts/observatory/control/constants/atcs_constants.py

```diff
@@ -0,0 +1 @@
+plate_scale = 0.01
```

Comment:
Can you a comment about the units? or, consider using satrapy units to define the variable.